ui/src/main/resources/index.html

```diff
@@ -52,5 +52,6 @@
       <!-- }}} -->
     </div>
     <script type="text/javascript" src="bundle.js"></script>
+    <script type="text/javascript" src="/admin/iframe-resizer/2.6.2/js/iframeResizer.contentWindow.min.js"></script>
```

Comment:
@pklinef @andrewkfiedler Is it okay if we reference the iframeresizer from the admin ui directly?